independent-publisher-connectors/Advanced Scraper/readme.md

```diff
@@ -0,0 +1,19 @@
+# Advanced Scraper
+An advanced web scraper API with rotating IPs from 170+ countries.
+
+## Publisher: Troy Taylor, Hitachi Solutions
+
+## Prerequisites
+The use of this API requires an account on [APILayer](https://apilayer.com/) and a subscription to the Advanced Scraper API.
+
+## Obtaining Credentials
+You can find you API key on your account page for APILayer.
```

Comment:
Hi @troystaylor, Thank you for the new submission again, you are doing an amazing job.
Can you please check out this small typo, I think you meant to say "You can find **your*** API key ....". Thank you very much.